org.uqbar.project.wollok.lib/src/wollok/lib/GameObject.xtend

```diff
@@ -67,6 +67,15 @@ class GameObject {
 		.toList.javaToWollok
 	}
 	
+	def colliders(WollokObject visual) {
+		val visualObject = board.findVisual(visual)
+		board.getComponentsInPosition(visualObject.position)
+		.map[ it as WVisual ]
+		.filter [ !it.equals(visualObject)]
+		.map [ it.wObject ]
+		.toList.javaToWollok
+	}
+		
```

Comment:
Should be cool, but I don't know if it's possible. (Bah, all is possible, but I don't know how many lines I have to touch. And I want merge this and two PRs more for v1.5.1 :P). Let me see.